crypto-primitives/src/crh/bowe_hopwood/mod.rs

```diff
@@ -149,6 +152,32 @@ impl<G: Group, W: PedersenWindow> FixedLengthCRH for BoweHopwoodPedersenCRH<G, W
                     .reduce(G::zero, |a, b| a + &b)
             })
             .reduce(G::zero, |a, b| a + &b);
+
```

Comment:
You can avoid duplication by doing `.sum()`; the latest master adds in support for this :)